lib/stm32/common/pwr_common_v1.c

```diff
@@ -166,11 +166,11 @@ void pwr_disable_wakeup_pin(void)
 The voltage detector threshold must be set when the power voltage detector is
 enabled, see @ref pwr_enable_power_voltage_detect.
 
-@returns boolean: TRUE if the power voltage is above the preset voltage
+@returns boolean: TRUE if the power voltage is below the preset voltage
 threshold.
 */
 
-bool pwr_voltage_high(void)
+bool pwr_voltage_low(void)
```

Comment:
Interesting. Is this something that should be sent upstream? (to the main libopencm3 repo)?

Also, instead of changing the name, it would be a better idea to invert the logic inside the function. Changing the name breaks the compatibility.

Reply:
Just flipping the logic inside works also, but the name won't make sense. 

pwr_voltage_high() will return true when voltage is low